### PR TITLE
[NF-28930] Create a build for ruby 2.3.8

### PR DESCRIPTION
**Pull request by BenKanouse**
This is the newest version of ruby 2.3.x. We plan to use it as the newest version in WebAdMIT.

### CODE REVIEW

**Comment by BenKanouse** [comment]
I think I should update the read me with the correct way to update this project.

File: README.md [hunk 1]
```diff
@@ -29,4 +29,18 @@
...
+
+Manually building a container can be done, but only if you have the correct rights on hub.docker.com
+
+```
+cd debian/jessie-2.3.8
+docker login username
+time docker build -t liaisonintl/ruby:debian-jessie-2.3.8 -f Dockerfile .
+time docker push liaisonintl/ruby:debian-jessie-2.3.8
+```
+There is also a manual build for the existing directories that is triggered every time that master is updated.
```
Comment (kstevens715) [inline README.md]: Should this say automated and not manual?

Comment (BenKanouse) [reply README.md]: D'oh... yeah ill change it now.

File: README.md [hunk 2]
```diff
@@ -29,4 +29,30 @@
...
+```
+cd debian/jessie-2.3.8
+docker login username
+time docker build -t liaisonintl/ruby:debian-jessie-2.3.8 -f Dockerfile .
+time docker push liaisonintl/ruby:debian-jessie-2.3.8
+```
+
+There is also an automated build for the existing directories that is triggered every time that master is updated.
+This is done with docker hub's built in building feature. Docs: https://docs.docker.com/docker-hub/builds/#create-an-automated-build.
+
```
Comment (pfcarrier) [inline README.md]: The idea is we leverage the docker hub to build the images for us.  this ensure that the image is "trusted" e.g. match the Dockerfile.  E.g. no one is having pull privileges on an automated build in docker hub, even admin

Comment (BenKanouse) [reply README.md]: OK, I'll change it around to show how to test it locally by building it but not show pushing it up.

**Pull request merged**